.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: Node.js CI with PostgreSQL
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build_and_test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: tbd
+          POSTGRES_DB: mockpfi
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    strategy:
+      matrix:
+        node-version: ['20.4.0']
```

Comment:
thoughts on using [`db/scripts/start-pg`](https://github.com/TBD54566975/tbdex-pfi-exemplar/blob/main/db/scripts/start-pg) to start a postgres container? e.g.

```yml
- name: Start Postgres
   run: ./db/scripts/start-pg
```

Reply:
can't use that inside a github action that that requires docker in docker which is ... a hole I have been down before.

Reply:
I don't want to be in that hole

Reply:
![image](https://github.com/TBD54566975/tbdex-pfi-exemplar/assets/14976/5889efa7-b626-4f67-9885-e6b7b7c99906)